pkg/controller/kfservice/resources/knative/configuration.go

```diff
@@ -37,6 +37,13 @@ const (
 	FrameworkConfigKeyName = "frameworks"
 )
 
+var ConfigurationAnnotationDisallowedList = map[string]struct{}{
```

Comment:
Better to store as a list? What would ever live in this object?

Reply:
To optimize on list traversal with O(1) lookup. Preoptimization